Week1/index.js

```diff
@@ -0,0 +1,245 @@
+'use-strict';
+
+const mysql = require('mysql');
+
+// Create connection
+const db = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'userdb',
+});
+db.connect(err => {
+  if (err) {
+    throw err;
+  }
+  console.log('MySql Connected...');
+});
+db.query('CREATE DATABASE IF NOT EXISTS world;', error => {
```

Comment:
You can use 'new_world' instead of 'world' here to use the data being loaded by world.sql